server/db/postgres/dataGenerator.js

```diff
@@ -37,22 +37,26 @@ log('symbol generation complete');
 //   find a random company name
 //   add element and company name to query string
 // send query
-var queriesArray = [];
+var companyArray = [];
 
 function batchCreate(start, batchSize) {
   var query = `INSERT INTO company (name, symbol) VALUES `;
   for (let i = start; i < (start + batchSize); i++) {
       var name = faker.company.companyName();
       query += `($$${name}$$,$$${tickers[i]}$$),`;
   }
-  queriesArray.push(query.substring(0, query.length-1) + `;`);
+  companyArray.push(query.substring(0, query.length-1) + `;`);
 }
 
 for (let i = 0; i < 11; i++) {
     batchCreate(i * batchSize,batchSize);
 }
 
-module.exports = { queriesArray };
+// for (var ele of tickers) {
+
+// }
```

Comment:
Is this not finished?  Want to make sure you didn't accidentally forget a piece of logic.